src/index.js

```diff
@@ -1,22 +1,21 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import { createStore, applyMiddleware } from "redux";
 import { Provider } from "react-redux";
-import createSagaMiddleware from "redux-saga";
-import { composeWithDevTools } from "redux-devtools-extension";
+import { buildRouter } from "@salsita/react-router";
+import { buildStore } from "@salsita/react-core";
 
 import { Root } from "modules/root/components/root";
 import { rootReducer } from "modules/root/root-reducer";
 import { rootSaga } from "modules/root/root-saga";
+import * as routes from "router/routes";
```

Comment:
It's common to use namespaces in pascal case and I would put it into `modules/` folder.
```suggestion
import * as Routes from "modules/router/routes";
```